core/Option.php

```diff
@@ -178,10 +178,6 @@ protected function getValue($name)
         $value = Db::fetchOne('SELECT option_value FROM `' . Common::prefixTable('option') . '` ' .
                               'WHERE option_name = ?', $name);
 
-        if ($value === false) {
```

Comment:
Initially I didn't get why there would be per request eg 140 calls to `Option::get()` from like 15 different places and 79 calls where actually hidding the DB. Until I realised for many of these values there is no value configured on `option` table and it would fetch the value from the DB again and again. Some option names it was fetching over 30 times.

Reply:
If a value is set later, it should go through ::set() anyway and be invalidated.